loops_lab.js

```diff
@@ -0,0 +1,316 @@
+// # Loops Lab
```

Comment:
Good demonstration of for and while loops overall. I like the use of ternary operators. Other than some spacing issues, everything looked fine. Good job!